Firestore/core/src/firebase/firestore/remote/serializer.cc

```diff
@@ -166,22 +249,38 @@ Writer Writer::Wrap(std::vector<uint8_t>* out_bytes) {
   return Writer(raw_stream);
 }
 
+Reader Reader::Wrap(const uint8_t* bytes, size_t length) {
+  pb_istream_t raw_stream = pb_istream_from_buffer(bytes, length);
```

Comment:
A few things:
  * clang-tidy will complain about "repeating the return type"
  * please use brace initialization if possible

Taken together: could this be
```
return {pb_istream_from_buffer(bytes, length)};
```

Reply:
Done, though as:
```
return Reader{pb_istream_from_buffer(bytes, length)};
```

(since ctor is explicit and lint otherwise complains.)